tests/rust/must-use.toml

```diff
@@ -0,0 +1,15 @@
+# Quite an abuse...
+autogen_warning = """
```

Comment:
Can we use the 'header' option for this?

Reply:
Yup! Done